packages/quantic/force-app/main/default/lwc/quanticFeedback/quanticFeedback.js

```diff
@@ -94,38 +94,38 @@ export default class QuanticFeedback extends LightningElement {
   _size = 'xx-small';
 
   /**
-   * Fires the "like" event.
+   * Fires the "quantic__like" event.
    * @returns {void}
    */
   handleLike() {
     this.dispatchEvent(
-      new CustomEvent(`like`, {
+      new CustomEvent(`quantic__like`, {
```

Comment:
These events haven't been updated where they were listened to:

Example
https://github.com/coveo/ui-kit/blob/a6dd19c883664e09a86dd80c81c55e637315ac8e/packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/templates/generatedAnswer.html#L14
https://github.com/coveo/ui-kit/blob/a6dd19c883664e09a86dd80c81c55e637315ac8e/packages/quantic/force-app/main/default/lwc/quanticSmartSnippet/quanticSmartSnippet.html#L31

Search for "onlike", "ondislike", "onpressexplainwhy"